packages/expo-cli/src/exit.ts

```diff
@@ -18,6 +18,7 @@ export function installExitHooks(
   }
 
   for (const signal of ['SIGINT', 'SIGTERM']) {
+    // @ts-ignore
```

Comment:
Whenever you add a ts-ignore directive, include a comment explaining why/what you are ignoring